### ISSUE TITLE
Works from local file but not when served from Exchange...

### ISSUE THREAD

**Issue by ascholnick** [tavicu/bs-confirmation#38]
Not sure if this is a bug or a configuration issue, but can anyone help me understand why the following code works fine when I load the page from a local filesystem file, but not once it is 'plugged in' to be served from my Exchange server.  When going through Exchange, I get a 'hover' tooltip popup showing the 'title' text, and a click on the button will immediately go to the associated link.  From the file - no tooltip, and the correct expected behavior (a box with buttons for "Looks Good" and "Not so much...").

Thanks! Here's the code:
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
 <html>
 <head>

<link href="https://netdna.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css" rel="stylesheet">
<link href="http://mistic100.github.io/Bootstrap-Confirmation/assets/css/docs.min.css" rel="stylesheet">
<link href="http://mistic100.github.io/Bootstrap-Confirmation/assets/css/style.css" rel="stylesheet">

<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"\>\</script\>
<script src="https://netdna.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"\>\</script\>
<script src="http://mistic100.github.io/Bootstrap-Confirmation/assets/js/docs.min.js"\>\</script\>

</HEAD>
<body>

<form id="special" action="" method="post" name="AddNewClient"></form>
<form id="normal" action="/FEdev/clientEntry.cgp" method="post" name="AddNewClient">
        <input name="cID" type="hidden" value="##cID##" form="normal"  />
        <input name="cID" type="hidden" value="" form="special"  />
        <center><button 
            class="btn btn-large btn-primary" 
            data-toggle="confirmation" 
            data-btn-ok-label="Looks Good" 
            data-btn-ok-icon="glyphicon glyphicon-share-alt" 
            data-btn-ok-class="btn-success" 
            data-btn-cancel-label="Not so much..." 
            data-btn-cancel-icon="glyphicon glyphicon-ban-circle" 
            data-btn-cancel-class="btn-danger"
            title="Are you certain that this client is unique?"
            data-placement="right"
            data-singleton="true"
            data-popout="true"  value="AddNewClient" name="DO_ME" form="normal" >ADD THE NEW ENTRY</button></center><br>
<hr style="height:1pt;visibility:hidden; margin-bottom:-10pt;"  />
            <h4><span style="display:inline-block; width: 25pt;"></span>
            <span style="display:inline-block; width: 25pt;"></span>
            <font color="#ff4444"><B>*</B></font>
            Company : <input maxlength="250" name="cCompany" size="80" type="text" value="" onsubmit="return chkClient(this)" form="normal" /><br>
Notes: <br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;  
<span style="display:inline-block; width: 60pt;"></span> 
<textarea cols="90" rows="10" name="cNotes" form="normal" ></textarea><br>
            <hr>
</form></font>
<script src="http://mistic100.github.io/Bootstrap-Confirmation/dist/bootstrap-confirmation2/bootstrap-confirmation.min.js"\>\</script\>
<script>
    $('[data-toggle=confirmation]').confirmation();
    $('[data-toggle=confirmation-singleton]').confirmation({ singleton:true });
    $('[data-toggle=confirmation-popout]').confirmation({ popout: true });
</script>
</body></html>

**Comment by ascholnick** [comment]
P.S. = problems persist across Chrome, Firefox, IE, and Opera browsers.

**Comment by tavicu** [comment]
Hi. You got the wrong repo :) http://mistic100.github.io/Bootstrap-Confirmation
